src/server/v2.0/handler/project.go

```diff
@@ -502,6 +505,75 @@ func (a *projectAPI) UpdateProject(ctx context.Context, params operation.UpdateP
 	return operation.NewUpdateProjectOK()
 }
 
+func (a *projectAPI) GetScannerOfProject(ctx context.Context, params operation.GetScannerOfProjectParams) middleware.Responder {
+	projectNameOrID := parseProjectNameOrID(params.ProjectNameOrID, params.XIsResourceName)
+	if err := a.RequireProjectAccess(ctx, projectNameOrID, rbac.ActionRead, rbac.ResourceScanner); err != nil {
+		return a.SendError(ctx, err)
+	}
+
+	p, err := a.projectCtl.Get(ctx, projectNameOrID, project.Metadata(false))
+	if err != nil {
+		return a.SendError(ctx, err)
+	}
+
+	scanner, err := a.scannerCtl.GetRegistrationByProject(ctx, p.ProjectID)
+	if err != nil {
+		return a.SendError(ctx, err)
+	}
+
+	return operation.NewGetScannerOfProjectOK().WithPayload(model.NewScannerRegistration(scanner).ToSwagger(ctx))
+}
+
+func (a *projectAPI) ListScannerCandidatesOfProject(ctx context.Context, params operation.ListScannerCandidatesOfProjectParams) middleware.Responder {
+	projectNameOrID := parseProjectNameOrID(params.ProjectNameOrID, params.XIsResourceName)
+	if err := a.RequireProjectAccess(ctx, projectNameOrID, rbac.ActionCreate, rbac.ResourceScanner); err != nil {
```

Comment:
ActionList?

Reply:
The action in the original API handler is `ActionCreate`.